docs/specification.yaml

```diff
@@ -767,6 +802,28 @@ components:
           example: "This is a _new_ page's content in **markdown**!"
       required:
         - route
+    FindPagesRequestBody:
+      type: object
+      properties:
+        taxonomyFilter:
+          type: array
+          items:
+            type: object
+            description: The taxonomy filters
+            properties:
+              custom_taxonomy_key:
+                type: string
+                example: "author"
+              custom_taxonomy_values:
+                type: array
+                items:
+                  type: string
+                  description: The taxonomy value
+                  example: "ksmith"
+        operation:
+          type: string
```

Comment:
Perhaps this should be an `enum` listing the specific allowed values?

Reply:
You're right